env-vars.list

```diff
@@ -0,0 +1,32 @@
+# Node related variables, change if you really want to!
+NODE_ENV=production
+NODE_PORT=3002
+
+# The secret used for generating JWTs
+JWT_SECRET=
+
+# The Steam API web key (you can get it from https://steamcommunity.com/dev/apikey)
+STEAM_WEB_API_KEY=
+
+# The database user, password, and host
+MOM_DATABASE_USER=
+MOM_DATABASE_PW=
+MOM_DATABASE_HOST=
+
+# The base URL of your website
+BASE_URL=
+
+# The Twitter consumer Key and Secret for using their API
+TWITTER_CONSUMER_KEY=
+TWITTER_CONSUMER_SECRET=
+
+# The Discord Client ID and Secret for using their API
+DISCORD_CLIENT_ID=
+DISCORD_CLIENT_SECRET=
+
+# The Twitch Client ID and Secret for using their API
+TWITCH_CLIENT_ID=
+TWITCH_CLIENT_SECRET=
+
+# A generated secret for express' sessions
+EXPRESS_SESSION_SECRET=
```

Comment:
How does this file work, do you need to make a copy of it or should the prod server just fill this in before running `runDocker.sh`?

Reply:
The `docker run` docker command mounts it with `--env-file env-vars.list`. So yep, you just need to fill it in before running.